tests/index.js

```diff
@@ -20,7 +20,7 @@ describe('application launch', function () {
 
   it('shows an initial window', function () {
     return this.app.client.getWindowCount().then(function (count) {
-      assert.equal(count, 1)
+      assert.equal(count, 2)
```

Comment:
Just curious about why the window count increased to 2? There doesn't seem an extra window displayed at start.